test_crates/workspace_overrides/current/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[workspace]
+resolver = "2"
+members = ["pkg"]
+
+[workspace.metadata.cargo-semver-checks]
```

Comment:
Might we want to add another layer of indirection here, like `workspace.metadata.cargo-semver-checks.lints`? With the current design, our entire metadata entry is reserved for lints and we can't add any other kind of config data at all.

Reply:
Sounds good.  I can open a separate PR for this.